cmd/win-sshproxy/main.go

```diff
@@ -100,12 +100,16 @@ func main() {
 	}
 
 	logrus.Debug("Setting up proxies")
-	setupProxies(ctx, group, sources, dests, identities)
+	err = setupProxies(ctx, group, sources, dests, identities)
+	if err != nil {
+		logrus.Errorf("Error setting up proxies: %v", err)
+		return
+	}
 
-	// Wait for cmopletion (cancellation) or error
+	// Wait for completion (cancellation) or error
 	if err := group.Wait(); err != nil {
-		logrus.Errorf("Error occured in execution group: " + err.Error())
-		os.Exit(1)
+		logrus.Errorf("Error occurred in execution group: " + err.Error())
+		return
```

Comment:
I find it easier to review lint fixes when similar fixes are split in their own commits as much as possible, and when the error message is part of the commit log. 
There could have been one commit about spell checking, one about error checks, and a last one for the few remaining lint issues (or multiple ones).

For this specific change, I had to rerun the linting myself to understand it:
```
cmd/win-sshproxy/main.go:108:3: exitAfterDefer: os.Exit will exit, and `defer log.Close()` will not run (gocritic)
		os.Exit(1)
```